src/settings.h

```diff
@@ -169,14 +169,21 @@ class Settings
         void setNotificationIcon(const QPixmap& icon);
 
     private:
+        // Loading from either storage
+        void    fromJSON( const QJsonObject& settings );
+
+        // TODO: remove this on March 23, 2022
```

Comment:
How about instead of this `TODO`, we have the following at the top of `fromSettings`, so we get a real reminder in two years:
```c++
static_assert(__DATE__[10] < '2', "Remove this function after 2022");
```

Reply:
No need for such drastic measures. If we forget about it because we're all hiding in bunkers from coronavirus, the code should still compile in 2023, and it won't be a big issue if it does not.